src/cmap/connection.ts

```diff
@@ -384,7 +384,8 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
       } else {
         // Get the first orphaned operation description.
         const entry = this[kQueue].entries().next();
-        if (entry) {
+        /* eslint no-restricted-syntax: 0 */
+        if (entry.value !== undefined) {
```

Comment:
What's the downside to checking for "not null" other than that we never expect it to be actually null?

Reply:
I never expected null, only undefined, but changing to check for not null actually works fine as well and does not need the eslint override. So I updated it.